src/plugins/vis_types/timelion/server/routes/run.ts

```diff
@@ -96,7 +95,7 @@ export function runRoute(
       });
       try {
         const chainRunner = chainRunnerFn(tlConfig);
-        const sheet = await Bluebird.all(chainRunner.processRequest(request.body));
+        const sheet = await Promise.all(await chainRunner.processRequest(request.body));
```

Comment:
`processRequest` returns a promise  that results in an iterable. Bluebird will handle that internally, but `Promise.all` only expects and iterable

Reply:
I saw this in the `processRequest` code and assumed it was a Promise returned: 
```js
return preProcessSheet(sheet).then(function () {
      return _.map(sheet, function (chainList, i) {
        return resolveChainList(chainList)
          .then(function (seriesList) {
            ...
          })
         ...
      });
    });
```

I already assumed I missed something here and approved to avoid to unblock.

Reply:
Yeah the `processRequest` could be improved the not return a promise as it's really not needed as it will always be resolved once returned. However, that's not in scope of this PR.

`Bluebird.all` is just a little more forgiving with what argument you give it and will detect if you give it a promise and automatically `await` it for you. With `Promise.all` I have to manually `await` before passing in the resolved value to the `all` method.